sdk/turing/generated/model/ensembler.py

```diff
@@ -62,7 +62,7 @@ class Ensembler(ModelComposed):
 
     validations = {
         ('name',): {
-            'max_length': 15,
+            'max_length': 20,
```

Comment:
Is it re-generated from openapi-specs or a manual fix? :)

Reply:
This is regenerated, did not manually modify this.

Reply:
Got it. Maybe we need to add it into our CI to check if the generated clients are in sync with the specs. (That's out of scope of this PR)